### DISCUSSION TITLE
Ignored/skipped tests are shown as Flaky

### DISCUSSION THREAD

**Discussion by lukasferda** [reportportal/reportportal#1570] [Q&A]
Hi, I am using ReportPortal.Nunit in C# E2E test project and _Flaky test cases table_ widget in Report portal. **The issue is that tests with Assert.Warn or Ignore are shown as flaky** but by my opinion should not be. 
Flaky test should be only Pass->Fail or Fail->Pass switch. 
Do you have same issue? Do you have any idea how to solve this?

**Comment by evjlobanova** [comment]
The widget is working as designed.  
Widget calculates a number of switches from one status to another.

**Comment by qasamppa** [comment]
I would say, that Skipped/Ignored cases calculation for flaky tests are not correct way, in my opinion. At least it should be configurable.

**Reply by chadcrum** [reply]
100% agree with this. I don't understand why skipped tests matter - some tests are expected to be skipped. I only want to know flaky pass/fail tests.

**Comment by mortenb-buypass** [comment]
This is till an issue
